pkg/reconciler/revision/resources/pa_test.go

```diff
@@ -203,6 +203,9 @@ func TestMakePA(t *testing.T) {
 					Namespace: "blah",
 					Name:      "batman",
 					UID:       "4321",
+					Labels: map[string]string{
+						serving.RoutingStateLabelKey: string(v1.RoutingStatePending),
+					},
```

Comment:
I'm not sure the test behavior is changed by this label (as the test passes both with and without it). Is it possible to add a test where we check the routing state of the revision?